tests/vm_test.go

```diff
@@ -428,6 +428,34 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			return updatedVM
 		}
 
+		table.DescribeTable("cpu/memory in requests/limits should allow", func(cpu, request string) {
+			vm := tests.NewRandomVirtualMachine(
+				tests.NewRandomVMIWithEphemeralDisk(
+					cd.ContainerDiskFor(cd.ContainerDiskCirros),
+				), false,
+			)
+			vm.APIVersion = "kubevirt.io/" + v1.ApiStorageVersion
+			vm.Spec.Template.Spec.Domain.Resources.Limits = make(k8sv1.ResourceList)
+			vm.Spec.Template.Spec.Domain.Resources.Requests[corev1.ResourceCPU] = resource.MustParse("222")
+			vm.Spec.Template.Spec.Domain.Resources.Limits[corev1.ResourceCPU] = resource.MustParse("222")
+			vm.Spec.Template.Spec.Domain.Resources.Requests[corev1.ResourceMemory] = resource.MustParse("2222222")
+			vm.Spec.Template.Spec.Domain.Resources.Limits[corev1.ResourceMemory] = resource.MustParse("2222222")
+
+			jsonBytes, err := json.Marshal(vm)
+			Expect(err).NotTo(HaveOccurred())
+
+			jsonString := strings.Replace(string(jsonBytes), "\"222\"", cpu, -1)
+			jsonString = strings.Replace(string(jsonBytes), "\"2222222\"", request, -1)
+
+			result := virtClient.RestClient().Post().Resource("virtualmachines").Namespace(tests.NamespaceTestDefault).Body([]byte(jsonString)).SetHeader("Content-Type", "application/json").Do()
```

Comment:
for peace of mind, would it be worth starting the VM with these requests and ensuring they make on the resulting pod in the way we expect?

Reply:
Well 
```
<domain type="kvm" xmlns:qemu="http://libvirt.org/schemas/domain/qemu/1.0">
	<name>kubevirt-test-default1_testvmi-699jp</name>
	<memory unit="b">0</memory>
```
that doesn't seem right for the float case.